src/javascript/SelectorTypes/SelectorTypes.js

```diff
@@ -50,7 +50,18 @@ export const registerSelectorTypes = ceRegistry => {
     ceRegistry.add('selectorType', 'Picker', {resolver: options => pickerConfigs.getPickerSelectorType(options)});
     registerPickerActions(ceRegistry);
 
-    ceRegistry.add('selectorType', 'Choicelist', {cmp: ChoiceList, supportMultiple: true});
+    ceRegistry.add('selectorType', 'Choicelist', {
+        cmp: ChoiceList,
+        supportMultiple: true,
+        initValue: field => {
+            const valueConstraints = field.valueConstraints.filter(v => v.properties.find(p => p.name === 'defaultProperty' && p.value === 'true'));
```

Comment:
You need to check if you are in create mode, because if for an existing node, the value has been removed, the default one will replace the empty one.